fastlane/lib/fastlane/actions/docs/sync_code_signing.md

```diff
@@ -322,9 +322,28 @@ You'll be asked for the new password on all your machines on the next run.
 If you want to manually decrypt a file you can.
 
 ```no-highlight
-openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d
+openssl aes-256-cbc -k <password> -in <fileYouWantToDecryptPath> -out <decryptedFilePath> -a -d
 ```
 
+### Export Distribution Certificate and Private Key as p12 (pkcs12) file
+
+The certificate is stores in the .cer file and the private key in .p12 file. They need to be decrypted and repackaged into a single encrypted p12 file.
+
+Decrypt your cert found in certs/distribution/<unique-id>.cer as a pem file
```

Comment:
I think we need to replace `/distribution` with `/<type>/` as the values could be `development`, `distribution` or `enterprise`